include/nudb/erase_file.hpp

```diff
@@ -0,0 +1,62 @@
+//
+// Copyright (c) 2015-2016 Vinnie Falco (vinnie dot falco at gmail dot com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+
+#ifndef NUDB_ERASE_FILE_HPP
```

Comment:
Of course I don't remember now.  But I'm guessing it was something to so with getting the documentation to come out write.  So I copied the pattern established with create, verify, vist, etc. each of which had their own file.
>